spec/schemas/ml._common.yaml

```diff
@@ -156,3 +156,131 @@ components:
     UndeployModelNodeStats:
       type: object
       additionalProperties: true
+    CreateConnectorRequest:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        version:
+          type: integer
+        protocol:
+          type: string
+        credential:
+          $ref: '#/components/schemas/Credential'
+        parameters:
+          $ref: '#/components/schemas/Parameters'
+        client_config:
+          $ref: '#/components/schemas/ClientConfig'
+        actions:
+          type: array
+          items:
+            $ref: '#/components/schemas/Action'
+      required:
+        - actions
+        - credential
+        - description
+        - name
+        - parameters
+        - protocol
+        - version
```

Comment:
This is exactly the data we're going to want to pull programmatically to validate our templates.  Glad to see it being added!